sdk/logical/events.go

```diff
@@ -5,10 +5,22 @@ package logical
 
 import (
 	"context"
+	"encoding/json"
+	"errors"
 
 	"github.com/hashicorp/go-uuid"
+	"google.golang.org/protobuf/types/known/structpb"
 )
 
+// common event metadata keys
+const (
+	EventMetadataSecretPath        = "secret_path"
```

Comment:
Looking at the KV PR, should this maybe be `api_path`, or `get_path` or something less specific to secrets? It's a bit of a misnomer for the config path, and similarly it doesn't make a ton of sense to me for auth plugins.